lib/rspec/support/differ.rb

```diff
@@ -2,6 +2,7 @@
 RSpec::Support.require_rspec_support 'hunk_generator'
 
 require 'pp'
+require 'rspec/support/object_inspector'
```

Comment:
This should use `RSpec::Support.require_rspec_support "object_inspector"`.  That uses `require_relative` on rubies that support that and normal `require` on rubies that don't.  We prefer `require_relative` when its available because it's always O(1) regardless of how large your load path where as `require` is O(N) based on the size of your load path.